projects/i18n/languages/italian/addon-editor.ts

```diff
@@ -0,0 +1,55 @@
+import {LanguageEditor} from '@taiga-ui/i18n/interfaces';
+
+export const TUI_ITALIAN_LANGUAGE_ADDON_EDITOR: LanguageEditor = {
+    colorSelectorModeNames: ['Tinta unita', 'Gradiente'],
+    toolbarTools: {
+        undo: 'Annulla',
+        redo: 'Rifai',
+        font: 'Font',
+        fontStyle: 'Stile font',
+        fontSize: 'Dimensione font',
+        bold: 'Grassetto',
+        italic: 'Corsivo',
+        underline: 'Sottolineato',
+        strikeThrough: 'Barrato',
+        justify: 'Giustifica',
+        justifyLeft: 'Allinea a sinistra',
+        justifyCenter: 'Allinea al centro',
+        justifyRight: 'Allinea a destra',
+        justifyFull: 'Giustifica pieno',
+        list: 'Lista',
+        indent: 'Rientro',
+        outdent: 'Elimina rientro',
+        unorderedList: 'Lista non ordinata',
+        orderedList: 'Lista ordinata',
+        quote: 'Virgolette',
+        foreColor: 'Colore',
+        backColor: 'Colore sfondo',
+        hiliteColor: 'Evidenziazione del testo',
+        clear: 'Pulisci',
+        link: 'Link',
+        attach: 'Allega file',
+        tex: 'Inserisci TeX',
+        code: 'Codice',
+        image: 'Inserisci image',
+        insertHorizontalRule: 'Inserisci riga orizzontale',
+        superscript: 'Apice',
+        subscript: 'Pedice',
+        insertTable: 'Inserisci tabella',
+        mergeCells: 'Unisci celle',
+        splitCells: 'Dividi celle',
+        rowsColumnsManaging: 'Gestione righe e colonne',
+        cellColor: 'Colore cella',
+    },
+    editorTableCommands: [
+        ['Inserisci colonna prima', 'Inserisci colonna dopo'],
+        ['Inserisci colonna prima', 'Inserisci colonna dopo'],
```

Comment:
Hi

Thanks for your PR!
I don't know italian but I suppose that there is an error here. 
In English this row:
```
['Insert row before', 'Insert row after'],
```

Reply:
Hi @nsbarsukov,
My bad! I'll fix it immediately! 
Thank you!